developer_manual/digging_deeper/translation.rst

```diff
@@ -111,7 +111,9 @@ For convenience, when your provider implements language detection, ``null`` will
 
 If your provider does not implement ``IDetectLanguageProvider``, ``$fromLanguage`` will never be null.
 
-The method ``detectLanguage`` takes a text in some language and outputs the code of that language, or ``null`` in case detection wasn't successful. The language code that this method returns should match the ones you return in ``getAvailableLanguages``.
+The method ``detectLanguage`` takes a text in some language and outputs the code of that language, or ``null`` in case detection wasn't successful. The language code that this method returns should be one of the languages returned in ``getAvailableLanguages``.
+
+should or has to be?
```

Comment:
that was a review question :)

Reply:
but it was inside the suggestion :D

Reply:
oh. right. :see_no_evil: 

It's should. It doesn't have to be.